src/egl/drivers/dri2/platform_android.c

```diff
@@ -1082,10 +1133,12 @@ droid_open_device(struct dri2_egl_display *dri2_dpy)
 {
    int fd = -1, err = -EINVAL;
 
-   if (dri2_dpy->gralloc->perform)
-         err = dri2_dpy->gralloc->perform(dri2_dpy->gralloc,
-                                          GRALLOC_MODULE_PERFORM_GET_DRM_FD,
-                                          &fd);
+   const gralloc_module_t *gralloc0;
+   gralloc0 = dri2_dpy->gralloc;
+
```

Comment:
You might also need to add equivalent support for https://github.com/android-ia/external-mesa/pull/44 with Gralloc 1.0